SourceS/sdl2_to_1_2_backports.h

```diff
@@ -873,3 +873,24 @@ inline char *SDL_GetPrefPath(const char *org, const char *app)
 
 	return retval;
 }
+
+inline char *SDL_itoa(int _Value, char *_Dest, int _Radix)

```

Comment:
This causes the Linux SDL 1.2 build to fail. Maybe this is just missing from the Amiga SDL?
It appears to have been present in SDL 1.2 for ages: https://github.com/spurious/SDL-mirror/blob/release-1.2.15/include/SDL_stdinc.h#L461

Reply:
This should probably just be `snprintf` anyway https://github.com/diasurgical/devilutionX/pull/557